static/js/compose.js

```diff
@@ -724,13 +724,49 @@ function validate_stream_message() {
     return true;
 }
 
+//The function will check whether the recipient is in settings.CROSS_REALM_BOT_EMAILS or settings.ADMIN_DOMAIN
 function validate_private_message() {
     if (exports.recipient() === "") {
         compose_error("Please specify at least one recipient", $("#private_message_recipient"));
         return false;
     }
+    else {
+        var private_recipients = exports.recipient().split(/\s*[,;]\s*/);
```

Comment:
Ahh, that makes sense.  Looks like we have the same code in the typeahead as well.  I think maybe we should refactor this to a function with a name like `util.extra_pm_recipients` (rather than hand-parsing in 3 places) so that at least this syntax is only set in one place.